MODDICORE-114: Add MARC-Instance default mappings for 880 fields .

VRohach opened the pull request:
## Purpose
Add MARC-Instance default mappings for 880 fields.
It based on the rules below:
   * Finds 880 field from the record. If exists, then retrieve first 3 digits from subfield '6'.
   * If there is value from this list: 100, 110, 111, 245 - then field-value '880' will be changed on 700,710,711,246 respectively.
   * Otherwise, '880' will be changed on the first 3-digits value.


## Approach

1. Processing for 880 field was added.
2. Tests added.
3. NEWS updated.

## Learning
   * More info: https://issues.folio.org/browse/MODDICORE-114

OleksiiKuzminov commented:
We don't want to hardcode some values to the mapper. Plz add additional fields into the default mapping JSON and improve the parser to read these additional params

OleksiiKuzminov reviewed (changes requested):
It is not a good approach to hardcode field names

VRohach commented:
> It is not a good approach to hardcode field names

@OleksiiKuzminov  I changed this mechanism and moved it to the 'rules.json', as you suggested. So, I added new field-rules for replacement logic to the 'rules.json'-file. 

Could you please review my improvements?
